static/elements/chromedash-featurelist.js

```diff
@@ -137,9 +140,20 @@ class ChromedashFeaturelist extends LitElement {
     }
   }
 
+  _calcFilteredWithState() {
```

Comment:
Looks like `this.filtered` is no longer used, and `_calcFilteredWithState` is called every `this.filtered` gets updated. Is it safe to just change the structure of `filtered` to be  `filteredWithState`, and use just one property?

Reply:
I was able to eliminate that as a property and just have openFeatures.